docker-entrypoint.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
-yarn forever start dist/services/scheduler.js
+yarn forever dist/services/scheduler.js
```

Comment:
this makes it so the process is not started in a daemon such that logs still get displayed